testsuite/docker/kafka/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM quay.io/strimzi/kafka:0.31.1-kafka-3.2.3
+FROM quay.io/strimzi/kafka:0.33.1-kafka-3.3.2
```

In case you want to use the latest ...

```suggestion
FROM quay.io/strimzi/kafka:0.33.2-kafka-3.4.0
```

---

I bumped this all around the examples and the testsuite, added a new profile and added it to Travis build.